lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.cpp

```diff
@@ -118,7 +118,10 @@ loadMatchingPDBFile(std::string exe_path, llvm::BumpPtrAllocator &allocator) {
   // executable.
   if (!FileSystem::Instance().Exists(pdb_file)) {
     const auto exe_dir = FileSpec(exe_path).CopyByRemovingLastPathComponent();
-    const auto pdb_name = FileSpec(pdb_file).GetFilename().GetCString();
+    const auto pdb_path_style =
+        FileSpec::GuessPathStyle(pdb_file).value_or(FileSpec::Style::native);
+    const auto pdb_name =
+        FileSpec(pdb_file, pdb_path_style).GetFilename().GetCString();
```

Comment:
This is required for the tests to run on non-Windows OSs. The executable contains the path to the PDB file. This is the absolute path as built on my machine. However, this is a Windows path with backslashes. `FileSpec` uses the native style as a default. On Windows, that works fine, but it breaks on other OSs.